src/RadioGroup/radioGroupStyles.ts

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import { RadioGroupProps } from './RadioGroup';
+
+interface StyledLabelProps {
+  disabled?: boolean;
+  checked?: boolean;
+}
+
+export const StyledLabel = styled.label<StyledLabelProps>(
```

Comment:
This could probably just be `ComponentPropsWithoutRef<'label'>` to let it take all label props and do away with the interface.